### ISSUE TITLE
questions about number of trainning epochs

### ISSUE THREAD

**Issue by gooners1886** [cypw/PyTorch-MFNet#10]
how many epochs do I need to train the Kinetics400 model using train_kinetics.py to get 72.8 accuary?
https://github.com/cypw/PyTorch-MFNet/blob/master/train_kinetics.py#L67   says 10000? 
Is this 10000 too large?

**Comment by cypw** [comment]
The training steps is shown [here](https://github.com/cypw/PyTorch-MFNet/blob/master/train_kinetics.py#L61) and [here](https://github.com/cypw/PyTorch-MFNet/blob/master/train_model.py#L101), i.e. about 100 epochs.

I set the `--end-epoch=10000` just because I want to manually terminate the training when validation accuracy saturated.
